### PR TITLE
Add analytic bijector

### LINKED ISSUE #6: Implement a C^k with k>2 coupling layer with analytic inverse.

### PR DESCRIPTION
**Pull request by Justinezgh**
This PR adds the elements to create the analytic bijector computed using sympy 

Closes #6.

### CODE REVIEW

File: sbiexpt/analytic_bijector.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    def _forward(self, x):
+      return f(x, self.a, self.b, self.c)
+
+    def _inverse(self, y):
+      return inv_f(y,self.a, self.b, self.c)
+
+    def _forward_log_det_jacobian(self, x):
+      grad = gradf(x,self.a,self.b,self.c)
+      return jnp.atleast_2d(jnp.log(jnp.abs(jnp.prod(grad, axis=1)))).T
```
Comment (EiffL) [inline sbiexpt/analytic_bijector.py:55]: hum that looks suspicious, the fact that you need a .T at the end.
Note that this may not be the most stable way of computing a log det in the general case (although here this should be fine). For stability we usually prefer to use this function: https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.linalg.slogdet.html

**Review by EiffL — CHANGES_REQUESTED** [review]
I think it looks good so far, but we should add unit tests to make sure things don't break and verify basic assumptions, like being invertible.

Once we have merged #10 , can you add similar tests? that will check the inverse at least, maybe the gradients also.

**Comment by EiffL** [comment]
So, what do you think @Justinezgh , do we keep this PR open? If the analytic formula doesnt work, maybe we can close this PR for now (the branch still exists)
